MediaTracker/app/src/main/java/com/izor066/android/mediatracker/ui/AddNewEntryManually.java

```diff
@@ -94,11 +97,21 @@ public void onDatePubSet(int datePub) {
     @Override
     public void onClick(View v) {
         if (v.getId() == R.id.btn_set_cover) {
-            mAddCover = addCover.getText().toString();
-            coverPreview.setVisibility(View.VISIBLE);
-            Picasso.with(this)
-                    .load(mAddCover)
-                    .into(coverPreview);
+            Log.e("WTF", addCover.getText().toString());
+            if (addCover.getText().length() == 0) {
```

Comment:
There's a little bit of repetition in these `if else`. Would it read better as below, since `mAddCover` is anyway initialised with `IMAGE_PLACEHOLDER`:

```java
if (addCover.getText().length() > 0) {
    mAddCover = addCover.getText().toString();
}

coverPreview.setVisibility(View.VISIBLE);
Picasso.with(this)
            .load(mAddCover)
            .into(coverPreview);
```